scripts/serverless.js

```diff
@@ -46,8 +52,33 @@ process.once('uncaughtException', (error) =>
     serverless,
     hasTelemetryBeenReported,
     commandUsage,
-  })
-);
+  });
+});
+
+process.on('SIGINT', () => {
+  clearTimeout(keepAliveTimer);
+  if (
+    commandSchema &&
+    !hasTelemetryBeenReported &&
+    !isTelemetryDisabled &&
+    (serverless ? serverless.isTelemetryReportedExternally : true)
+  ) {
+    const telemetryPayload = generateTelemetryPayload({
+      command,
+      options,
+      commandSchema,
+      serviceDir,
+      configuration,
+      serverless,
+      commandUsage,
+    });
+    const failureReason = {
```

Comment:
I think we should not add it, this data feels suprefluos (it's only errors where we add additional distribution by _kind_(